buildroot-external/rootfs-overlay/usr/bin/ha

```diff
@@ -3,4 +3,9 @@
 # HA utility
 # ==============================================================================
 
-docker exec -it hassio_cli ha "$@"
+if [ -t 1 ]; then
+	# stdout (fd 1) must be terminal, otherwise `docker exec -ti` fails
+	docker exec -it hassio_cli ha "$@"
+else
+	docker exec hassio_cli ha "$@"
```

Comment:
Should this be `-i` as well? We did introduce this for `datadisk wipe` command, which was the first command reading from stdin (to ask for conformation). Now to fix the issue at hand, `-t` is problematic. But maybe we should still use `-i` here?
```suggestion
	docker exec -i hassio_cli ha "$@"
```

Reply:
It depends... We might actually not even need `-t`, we don't do anything that will need terminal - like interactive completion, echo control, etc., we just won't show the spinner: https://github.com/home-assistant/cli/blob/713510d4333b0bb25d8d0b677d5369bd73861f7c/cmd/root.go#L47C1-L51C4

If we add `-i` also to the "no terminal" variant, calling anything that needs to read from stdin (like the datadisk wipe) without writing anything to the input will cause the script to hang indefinitely. If we don't attach stdin, it will just read EOF and do nothing. So it depends what behavior is better, I might prefer the current one.

Reply:
Fair enough, I added it and adjusted the comment.